test/cell_test.rb

```diff
@@ -0,0 +1,113 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/ship'
+require './lib/cell'
+
+class CellTest < Minitest::Test
+
+  def setup
+    skip
+    @cell = Cell.new("B4")
+  end
+
+  def test_it_exists
+    skip
+    assert_instance_of Cell, @cell
+  end
+
+  def test_it_has_coordinates
+    skip
+    assert_equal "B4", @cell.coordinate
+  end
+
+  def test_it_does_not_have_a_ship
+    skip
+    assert_equal nil, @cell.ship
+  end
+
+  def test_it_is_empty
+    skip
+    assert @cell.empty?
+  end
+
+  def test_it_has_a_new_ship
+    skip
+    assert_equal @cruiser, @cell.place_ship
+  end
+
+  def test_it_is_not_empty
+    skip
+    refute @cell.empty?
+  end
+
+  def test_it_has_not_been_fired_upon
+    skip
+    refute @cell.fired_upon?
+  end
+
+  def test_when_hit_loses_1_health
+    @cruiser.hit
+    assert_equal 2, @cell.ship.health
+  end
+
+  def test_it_has_been_fired_upon
+    skip
+    assert @cell.fired_upon?
+  end
+
+  def test_cell_has_not_been_fired_upon
+    skip
+    assert_equal ".", cell_1.render
+  end
+
+  def test_fired_upon_and_shot_missed
+    skip
+    assert_equal "M", cell_1.render
+  end
+
+  def test_it_has_a_new_cell
+    skip
+    assert_equal cell_2, Cell.new("C3")
+  end
+
+  def test_cell_has_a_ship
+    skip
```

Comment:
@amymoller 
So on this one you're going to want to first define `cell_2` and `cruiser`.
 
**Tip:** If you want to stop having to define `cruiser` , `cell_1`, `cell_2` and so forth repeatedly, you can add these to your `setup` method at the very beginning. 
Then you can access these variables with `@` notation like you're doing below.